apps/services/handlers.py

```diff
@@ -388,6 +391,46 @@ async def __generate_user_document_name_for_download(
             f"{student.last_name}.docx"
         )
 
+    @classmethod
+    async def __create_user_warrant(
+        cls, session: AsyncSession, data
+    ) -> UserDocument:
+        content = await cls.__create_user_warrant_content(data)
+        user_document_record = await user_document_service.create(
```

Comment:
we don't need to create a new record, just generate file and return it - that's all